lib/dispatch-rider/logging/lifecycle_logger.rb

```diff
@@ -11,8 +11,8 @@ def log_got_stop(reason, message)
         end
 
         def wrap_handling(message)
-          log_start(message)
           start_time = Time.now
+          log_start(message)
```

Comment:
if `log_start` fails, the `start_time` won't get initialized, but we are using it in an `ensure` block.